src/modules/ROOT/pages/changelog/202309/04.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta": {
+    "releasedOn": "2024.02.26"
+  },
+
+  "changed": [
+    {
+      "what": [
+        "asterisk color of record field from blue to red."
+      ],
+      "why": [
+        "to make the required fields stand out."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/65832b461c7bb88df4648242f102b4a45ad512ca"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "incorrect value applying to date and time record field when trying to remove an already set value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/6fe0b8ae83121d5e4d93ddb6a5b7b1778b8730d9"
+      ]
+    },
+    {
+      "what": [
+        "unnecessary showing of stale data warning when viewing a record page."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/070e7516bd72b3f1e340dfeb9890ffb8808e84f4"
+      ]
+    },
+    {
+      "what": [
+        "{PRODUCT_NAME} refusing to start when upgrading with alterations to the default {APP_NAME_COMPOSE} records schema."
+      ],
+      "how": [
+        "by allowing alterations to be logged for the default schema but continue blocking them from being applied."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/b389d5039d868be4573af012a813e7ef8bc90f81"
+      ]
+    },
+    {
+      "what": [
+        "page layout loading multiple times and always redirecting to the current layout after a record was saved."
```

Comment:
this also caused an error where it'd prompt an alert that no layout matches where in fact there were some